modules/users/tests/server/user.server.routes.tests.js

```diff
@@ -101,6 +153,110 @@ describe('User CRUD tests', function () {
     });
   });
 
+  it('should be able to get a single user details if admin', function (done) {
+    user.roles = ['user', 'admin'];
+
+    user.save(function () {
+      agent.post('/api/auth/signin')
+        .send(credentials)
+        .expect(200)
+        .end(function (signinErr, signinRes) {
+          // Handle signin error
+          if (signinErr) {
+            return done(signinErr);
+          }
+
+          // Get single user information from the database
+          agent.get('/api/users/' + user._id)
+            .expect(200)
+            .end(function (userInfoErr, userInfoRes) {
+              if (userInfoErr) {
+                return done(userInfoErr);
+              }
+
+              userInfoRes.body.should.be.instanceof(Object);
+              userInfoRes.body._id.should.be.equal(String(user._id));
+
+              // Call the assertion callback
+              done();
```

Comment:
the test anyway ends here so it doesn't matter that much, right?
I anyway updated to return here so it is consistent with the rest of the tests